### PR TITLE
Duplicate Interests Fix

### PR DESCRIPTION
**Pull request by pnor**
## Overview

Fixed bug in onboarding where way more than the expected amount of cells would show up.
Also fixed the search bug that would crash the app.

## Changes Made

Mostly was small one line changes that fixed bugs. Usually wrong values for tableView delegate, or wrong array being accessed.

## Test Coverage

iOS simulator

## Related PRs or Issues

#44 

## Screenshots

<details>

  <summary>Demo</summary>

  
  demo

</details>

### CODE REVIEW

**Review by lucyxubroad — APPROVED** [review]
Yay! glad this was a simple fix :)

File: CoffeeChat/Controller/GroupsViewController.swift [hunk 1]
```diff
@@ -256,11 +257,12 @@ extension GroupsViewController: UITableViewDataSource {
     }
 
     func numberOfSections(in tableView: UITableView) -> Int {
-        return displayedGroups.count
+        return 1
```
Comment (ywsang) [inline CoffeeChat/Controller/GroupsViewController.swift:260]: I _think_ 1 is default for number of sections

File: CoffeeChat/Controller/OnboardingInterestsViewController.swift [hunk 2]
```diff
@@ -162,7 +162,7 @@ extension OnboardingInterestsViewController: UITableViewDataSource {
     }
 
     func numberOfSections(in tableView: UITableView) -> Int {
-        return interests.count
+        return 1
```
Comment (ywsang) [inline CoffeeChat/Controller/OnboardingInterestsViewController.swift:165]: same thing here

**Review by ywsang — APPROVED** [review]
one small thing, otherwise lgtm!

**Pull request merged**